frontend/src/components/UserSearch.vue

```diff
@@ -132,7 +132,19 @@
           :loading="userSearchLoadingStatus"
           loading-text="Searching for users"
           v-on:click:row="selectUser"
-        ></v-data-table>
+        >
+          <template v-if="searchResults.length > 0" v-slot:footer>
+            <v-toolbar flat>
+              <v-spacer/>
+              <download-csv
+                  :data="searchResults"
+                  :fields="['id', 'username', 'enabled', 'firstName', 'lastName', 'email']"
```

Comment:
I think this will be easy to work with and extend. If we need roles, we just need to get that data into the `searchResults` and figure out how to make `fields` dynamic.

Alternatively, make a copy of `searchResults` and modify it however we want, then don't use `fields` at all.